src/stream/src/executor/managed_state/aggregation/extreme.rs

```diff
@@ -317,42 +357,26 @@ where
     }
 
     /// Flush the internal state to a write batch.
-    fn flush_inner(&mut self, write_batch: &mut WriteBatch<S>) -> StreamExecutorResult<()> {
```

Comment:
The flush now becomes very clean. That's why I think maybe in future we should remove `flush` on ManagedState

Reply:
Add a `TODO` here? The original doc seems confusing now. 😄